.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+- **Please check if the PR fulfills these requirements**
+  - [ ] Tests for the changes have been added (for bug fixes/features)
+  - [ ] Docs have been added / updated (for bug fixes / features)
```

Comment:
What does this question mean, i.e. what is meant by `Docs`?

Reply:
Me as a developer just would not know when to tick the box because we don't have any guideline on adding documentation...

Reply:
Shall I remove it then?

Reply:
What is your oppinion?

Reply:
I'll remove it.